### ISSUE TITLE
WESTERN ISO 8859-1 encoding not working once upgading to Atom 1.17.0

### ISSUE THREAD

**Issue by CaptainCapstone** [atom/atom#14566]
After upgrading to Atom 1.17.0, file encoding for Western ISO 8859-1 is broke.  Characters that used to show correctly are now replaced with �.  

So, at Atom 1.16.0, a line of code would look like this:
IF VAR1 ¬= VAR2 THEN
Once upgrading to 1.17.0, we now see:
IF VAR1 �= VAR2 THEN

Other editors show the code correctly with the codepage set to ISO 8859-1.

**Comment by CaptainCapstone** [comment]
Note: It doesn't seem to matter what file encoding I pick, even Chinese, the text in Atom never changes.  It appears the default of UTF-8 is always used, no matter what the user picks.  This is just an assumption, so please treat it as such :)

**Comment by CaptainCapstone** [comment]
Built-in Atom Packages (90)
├── atom-dark-syntax@0.28.0
├── atom-dark-ui@0.53.0
├── atom-light-syntax@0.29.0
├── atom-light-ui@0.46.0
├── base16-tomorrow-dark-theme@1.5.0
├── base16-tomorrow-light-theme@1.5.0
├── one-dark-ui@1.10.3
├── one-light-ui@1.10.3
├── one-dark-syntax@1.7.1
├── one-light-syntax@1.7.1
├── solarized-dark-syntax@1.1.2
├── solarized-light-syntax@1.1.2
├── about@1.7.6
├── archive-view@0.63.2
├── autocomplete-atom-api@0.10.1
├── autocomplete-css@0.16.1
├── autocomplete-html@0.7.3
├── autocomplete-plus@2.35.3
├── autocomplete-snippets@1.11.0
├── autoflow@0.29.0
├── autosave@0.24.2
├── background-tips@0.27.0
├── bookmarks@0.44.3
├── bracket-matcher@0.85.5
├── command-palette@0.40.4
├── dalek@0.2.1
├── deprecation-cop@0.56.7
├── dev-live-reload@0.47.1
├── encoding-selector@0.23.3
├── exception-reporting@0.41.3
├── find-and-replace@0.207.6
├── fuzzy-finder@1.5.6
├── git-diff@1.3.5
├── go-to-line@0.32.0
├── grammar-selector@0.49.4
├── image-view@0.61.2
├── incompatible-packages@0.27.3
├── keybinding-resolver@0.38.0
├── line-ending-selector@0.6.3
├── link@0.31.3
├── markdown-preview@0.159.11
├── metrics@1.2.3
├── notifications@0.67.1
├── open-on-github@1.2.1
├── package-generator@1.1.1
├── settings-view@0.249.4
├── snippets@1.1.4
├── spell-check@0.71.4
├── status-bar@1.8.7
├── styleguide@0.49.6
├── symbols-view@0.115.5
├── tabs@0.106.0
├── timecop@0.36.0
├── tree-view@0.217.0-7
├── update-package-dependencies@0.11.0
├── welcome@0.36.2
├── whitespace@0.36.2
├── wrap-guide@0.40.1
├── language-c@0.57.0
├── language-clojure@0.22.2
├── language-coffee-script@0.48.6
├── language-csharp@0.14.2
├── language-css@0.42.2
├── language-gfm@0.88.1
├── language-git@0.19.0
├── language-go@0.43.1
├── language-html@0.47.2
├── language-hyperlink@0.16.1
├── language-java@0.27.0
├── language-javascript@0.126.1
├── language-json@0.19.0
├── language-less@0.32.0
├── language-make@0.22.3
├── language-mustache@0.13.1
├── language-objective-c@0.15.1
├── language-perl@0.37.0
├── language-php@0.38.0
├── language-property-list@0.9.1
├── language-python@0.45.2
├── language-ruby@0.71.0
├── language-ruby-on-rails@0.25.2
├── language-sass@0.59.0
├── language-shellscript@0.25.0
├── language-source@0.9.0
├── language-sql@0.25.4
├── language-text@0.7.2
├── language-todo@0.29.1
├── language-toml@0.18.1
├── language-xml@0.35.0
└── language-yaml@0.29.0

Community Packages (14)  /Users/xxxxxxxx/.atom/packages
├── atom-overtype-mode@0.3.4
├── auto-detect-indentation@1.3.0
├── git-control@0.8.2
├── git-plus@7.3.3
├── git-time-machine@1.5.9
├── hacking-the-kernel@1.6.2
├── hacking-the-kernel-ui@1.6.2
├── highlight-selected@0.13.1
├── language-rexx@0.0.0
├── minimap@4.28.2
├── minimap-highlight-selected@4.6.1
├── open-recent@5.0.0
├── split-diff@1.4.0
└── todo-show@1.11.0

**Comment by CaptainCapstone** [comment]
```
apm      1.17.0
npm      3.10.5
node     6.9.5
python   2.7.10
git      2.11.0

```

**Comment by 50Wliu** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of atom/encoding-selector#48 and will be fixed in the next hotfix release, Atom 1.17.1. You may want to subscribe there for updates.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
